src/transitions/Slide.spec.js

```diff
@@ -181,4 +182,37 @@ describe('<Slide />', () => {
       });
     });
   });
+
+  describe('slidemount', () => {
+    let mount;
+    let mountWrapper;
+    let instance;
+
+    before(() => {
+      mount = createMount();
+      mountWrapper = mount(<Slide><div /></Slide>);
+    });
+
+    after(() => {
+      mount.cleanUp();
+    });
+
+    it('default value', () => {
+      mountWrapper.setProps({ in: false });
+      instance = mountWrapper.instance();
+      instance.componentDidMount();
+      assert.strictEqual(
+        ReactDOM.findDOMNode(mountWrapper.instance().transition).style.transform,
```

Comment:
What about ignoring that issue? It's testing after all, how cares to be safe?
We could create a `$FlowIgnore` type for that case: https://flow.org/en/docs/config/options/#toc-suppress-type-string.